### PR TITLE
Math magicians: full website

### PR DESCRIPTION
**Pull request by butlermuwo**
## Math magicians: full website
In this pull request I did the following:

✅ Created a website consisting of 3 pages: Home, Calculator, and Quote.
✅ Kept the general layout to match the wireframes.
✅ Added my own styles to improve the look and feel of the app.
✅ I created a route for every page.
✅ The navigation links for all 3 routes are displayed on each page.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
+/* eslint-disable import/extensions */
 
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-  }
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import Calculator from './components/Calculator';
```
Comment (SajjadAhmad14) [inline src/App.js:3]: There is a compilation error in `App.js` which stops the app to run. Kindly fix the `./components/Calculator` to `./components/calculator`.
error

File: src/App.js [hunk 2]
```diff
@@ -1,19 +1,22 @@
-import React from 'react';
-import Calculator from './components/calculator';
+/* eslint-disable react/jsx-key */
+/* eslint-disable import/extensions */
```
Comment (SajjadAhmad14) [inline src/App.js]: Please do not disable linters in every component and fix errors raised by linters instead.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Butler, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
README is professional. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [x] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: .eslintrc.json [hunk 3]
```diff
@@ -1,27 +1,26 @@
...
-    ]
-  }
\ No newline at end of file
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "rules": {
+    // "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
+    // "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
+    "react/react-in-jsx-scope": "off",
```
Comment (Cathella) [inline .eslintrc.json]: Hello, @butlermuwo, it is not good practice editing the linter files to your liking. I suggest that we mantain the rules set and try to improve the code base.

- [x] Please kindly make the required changelles

**Review by Cathella — APPROVED** [review]
## 🟡 Status: Changes Required 🛑

### Hi @butlermuwo 👋🏻, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 2 above)
Comment (butlermuwo) [reply src/App.js]: Hey Sajjad! I have tried working on this error but I can't see it locally and it's only appearing on GitHub kindly assist me to fix it. Additionally, I have tried looking for answers online and from folks at microverse and we have all failed.

File: .eslintrc.json (hunk 3 above)
Comment (butlermuwo) [reply .eslintrc.json]: Hi Cathella! I have maintained the codebase as you suggested, when I run eslint locally the linters test is passing except on Github kindly help me solve this, I have tried reaching out for help and I have done some research online but I can't get conclusive responses to slove this problem.

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @butlermuwo 👋🏻, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**